content/millennium/dstu2/infrastructure/document-reference.md

```diff
@@ -226,14 +225,22 @@ _Implementation Notes_
  Name     | Required? | Type          | Description
 ----------|-----------|---------------|-------------------------------------------------
 `patient` | Y         | [`reference`] | A reference to the patient whose document references are required. Example: `12345`
-`type`    | Y         | [`token`]     | The document reference type, can be a list of comma separated values. Example: `http://loinc.org|34133-9`
-`start`   | N         | [`date`]      | The start of the date range from which document reference records should be included. If not provided, then all records from the beginning of time are included. Example: `2014-09-24T12:00:00.000Z`
-`end`     | N         | [`date`]      | The end of the date range till which document reference records should be included. If not provided, then all records up to the current date are included. Example: `2016-09-24T12:00:00.000Z`
+`type`    | Y         | [`token`]     | The document reference type. Example: `http://loinc.org\|34133-9`
+`start`   | N         | [`date`]      | The start of the date range from which document reference records should be included. Example: `2014-09-24T12:00:00.000Z`
+`end`     | N         | [`date`]      | The end of the date range till which document reference records should be included. Example: `2016-09-24T12:00:00.000Z`
+
+_Implementation Notes_
 
-Notes:
+- The `type` parameter:
+  - It must include both a system and a code. (e.g. `&type=http://loinc.org\|34133-9`)
+  - It may be a single system and code, or a comma-separated list.
+- The `start` and `end` parameters:
+  - They must be valid [dateTime]s with a time component.
+  - They must have prefixes of `eq` or nothing.
+  - If `start` is not provided, then all records from the beginning of time are included.
+  - If `end` is not provided, then all records up to the current date are included.
```

Comment:
happen to know if this is inclusive? is it to the current date or dateTime?

my guess is inclusive to the current dateTime, but I haven't fully tested it

Reply:
Inclusive to current date.  implied ge / le prefixes